sources/packages/api/src-sql/Reports/Create-disbursement-report.sql

```diff
@@ -0,0 +1,44 @@
+--Insert config for disbursement report.
+INSERT INTO
+    sims.report_configs (report_name, report_sql)
+VALUES
+    (
+        'Disbursement_Report',
+        'SELECT "Date of Disbursement",
+            "SIN",
+            "Application Number",
+            "Certificate Number",
+            "Funding Code",
+            "Disbursement amount"
+          FROM
+            (SELECT TO_CHAR(DR.DISBURSE_DATE, ''YYYY-MM-DD'') AS "Date of Disbursement",
```

Comment:
I missed it in the previous PR, it is the double quotes on  `''YYYY-MM-DD''` needed because of Typeorm?
@dheepak-aot

Reply:
Yes @andrewsignori-aot it translates to single quotes, when the query is inserted during migration.